docs/user/exams/instructors_guide.rst

```diff
@@ -70,7 +81,32 @@ During the exam creation and configuration, you can create your exam and configu
 
    Exam Assessment and Student Review
 
-    .. _exercise_groups:
+
+
+**1.2.3 Import Exam**
+
+- Alternatively, you can import an existing exam by clicking on |import_exam|.
+- Artemis displays a list of all available exams. To select one specific exam for the import, click on the |import_individual_exam| button.
+
+.. figure:: instructor/exam_import_selection.png
+   :alt: List with all exams available for import
+   :align: center
+
+- You are now presented with the *Import Exam* view. All information except for the dates are copied from the exam, you selected for the import. You can find more informations regarding this view at the section :ref:`create and configure an exam <_configure_exam>`.
+- Additionally, you can select or deselect exercises, which are imported alongside the exam.
+- You can change the `title` and `isMandatory` of an exercise group, as well as the `title` (and `short-name` for programming exercises) for the individual exercises.
```

Comment:
```suggestion
- You can change the ``title`` and ``isMandatory`` of an exercise group, as well as the ``title`` (and ``short-name`` for programming exercises) for the individual exercises.
```
*Italics* vs ``monospace``? In 1.2.2 monospace is used. It should be made consistent in one style.